Makefile

```diff
@@ -8,6 +8,9 @@ OTELCOL_BUILDER ?= ${OTELCOL_BUILDER_DIR}/ocb
 DISTRIBUTIONS ?= "otelcol,otelcol-contrib,otelcol-k8s"
 GEN_CONFIG_DISTRIBUTIONS ?= "otelcol,otelcol-contrib"
 
+# required to use specific ocb builder version
+GO111MODULE=on
```

Comment:
Isnt that deprecated? AFAIK it should no longer has any effect.

Reply:
hmm, it was failing on the workflow run without it. I can give it a try without in my own repo and see

Reply:
yeah not sure why I was getting that error; not happening anymore. I'll remove this variable.